### DISCUSSION TITLE
How to reconnect to external jack clients?

### DISCUSSION THREAD

**Discussion by oleg68** [falkTX/Carla#1419] [Q&A]
Hello. I'm new in Carla and I'm migrating to it from qjackctl.

I have some troubles with Carla connection feature. 

I can create a connection graph with carla and save it. But seams Carla prrocesses it and makes connections only when the project is opening. If an external application restarts, it becomes disconnected. I need to go to the `File`->`Open` menu and to select the project file again for reconnecting.

Qjackctl had an `active patchbay` feature that was scanning for the running jack clients and automaticaly connected them along the patchbay rules. Is there some feature in Carla?

Is there a simple way of recconnecting without lots of mouse clicks. For example, any `Reopen`  button in Carla?

**Comment by falkTX** [comment]
You might want to use a session manager for that.
There are a few still maintained, for example https://github.com/jackaudio/new-session-manager and https://www.laborejo.org/agordejo/
